account/init_tenant.go

```diff
@@ -95,7 +95,7 @@ func CleanTenant(ctx context.Context, config tenantConfig, remove bool) error {
 	}
 	defer rest.CloseResponse(res)
 
-	if res.StatusCode != http.StatusOK {
+	if res.StatusCode != http.StatusNoContent { // the delete tenant operation returns a `204 No Content` response in case of success
```

Comment:
Check for Success range instead of specific code: >= 200 && < 300

Reply:
sure, that will be safer

Reply:
@aslakknutsen done in b27bf89